<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BatteryForm</class>
 <widget class="QWidget" name="BatteryForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QComboBox { min-height: 25px; min-width: 40px; }
QComboBox QAbstractItemView::item { min-height: 40px; min-width: 60px; }
</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>480</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>701</width>
      <height>361</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox{
  min-height: 28px; min-width: 40px;
/*  background-color: rgb(0, 170,0);*/
  background-color: rgb(12, 148, 214);

}

QComboBox QAbstractItemView::item {
  min-height: 35px; min-width: 60px;
 /* background-color: rgb(0, 170,0);
  background-color: rgb(75, 190, 155);*/

}</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>701</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="title">
      <string>电池单体数据</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="2">
       <widget class="QLabel" name="label_16">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>最低电压:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QLabel" name="label_18">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string> 查询组数：</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_15">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>最高电压:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QComboBox" name="betterypack_choice">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="lineEdit_H_Voltager">
        <property name="font">
         <font>
          <family>Mallige</family>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="lineEdit_L_Voltager">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>89</y>
       <width>701</width>
       <height>281</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>WenQuanYi Micro Hei</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="cursor" stdset="0">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::ActionsContextMenu</enum>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="dragEnabled">
      <bool>false</bool>
     </property>
     <property name="textElideMode">
      <enum>Qt::ElideMiddle</enum>
     </property>
     <property name="verticalScrollMode">
      <enum>QAbstractItemView::ScrollPerItem</enum>
     </property>
     <column>
      <property name="text">
       <string>电池序号</string>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>单体电压(V)</string>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>电池序号</string>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>单体电压(V)</string>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>电池序号</string>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>单体电压(V)</string>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>274</x>
      <y>0</y>
      <width>250</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>                     电池信息</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_homepage">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>418</y>
      <width>85</width>
      <height>38</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>WenQuanYi Micro Hei</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>首页</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_uppage">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>418</y>
      <width>85</width>
      <height>38</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>WenQuanYi Micro Hei</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>上一页</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_nextpage">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>418</y>
      <width>85</width>
      <height>38</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>WenQuanYi Micro Hei</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>下一页</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_endpage">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>418</y>
      <width>85</width>
      <height>38</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>WenQuanYi Micro Hei</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>末页</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
